src/revivalpmmp/pureentities/PluginConfiguration.php

```diff
@@ -45,7 +45,7 @@ class PluginConfiguration {
     private $pickupLootTicks = 10; // default: 10
     private $logEnabled = false; // enable or disable file logging
     private $enableSpawning = true; // enable spawning of entities
-    private $enabledWorlds = [];
+    private $enabledWorlds = []; // worlds where spawning of entities is allowed
```

Comment:
This feature is apart of the PureEntitiesX Protection plugin

Reply:
A different plugin?!

Reply:
lmao didn't know it was a thing did ya?

Reply:
I do now, but why 2 plugins if its only job is extra functionality for PureEntitiesX?